src/package/package_api.js

```diff
@@ -84,7 +84,7 @@ function defineReadOnlyProperty(object, key, value) {
 function PackageInformation(json) {
   for (var field in json) {
     var val = json[field];
-    if (field === 'installDate')
+    if (field === 'lastModified')
```

Comment:
Mybe it'a mistake at the beginning,there is no installDate in PackageInformation (https://developer.tizen.org/dev-guide/2.2.1/org.tizen.web.device.apireference/tizen/package.html#::Package::PackageInformation)
see also ````src/package/package_info.cc````